chore: v1.0.0-beta.130 (#1151)
Redocly/redocly-cli: docs/changelog.md

# Redocly CLI changelog

## 1.0.0-beta.130 (2023-07-?)

### Features

> - Added new rule `component-name-unique` to check for unique component names

Reviewer: Do we thank external contributors? Can we?

Reviewer 2: ```suggestion
- Added new rule `component-name-unique` to check for unique component names (kudos to @pboos 🎉).
```

# Redocly CLI changelog

## 1.0.0-beta.130 (2023-07-06)

### Features

- Added new rule `component-name-unique` to check for unique component names (kudos to @pboos 🎉).

> - Added docker container to GitHub packages.

Reviewer: ```suggestion
- Added docker container to GitHub container registry.
```

### Fixes

- Fixed an issue when the `--files` option of the `push` command did not upload extra files.

> - Fixed wrong schema type validation.

Reviewer: What did we fix here? Was this the null validation thing? Can we expand a bit please

Reviewer 2: @lornajane how about this?
```suggestion
- Fixed nullable schema type validation for OAS 3.1.
```

> - Fixed issue with broken styles in `build-docs` command.

Reviewer: Was this the additional peer dependencies change? I think it might be worth saying something like "Added peer dependencies to fix an issue with styles seen in the `build-docs` command.

Reviewer 2: ```suggestion
- Added peer dependencies to fix an issue with styles seen in the build-docs command.
```

### Changes

> - Improved data obfuscation.

Reviewer: ```suggestion
- Additional data masking for the anonymous data collection feature.
```
